tee-worker/enclave-runtime/src/rpc/common_api.rs

```diff
@@ -233,8 +233,18 @@ pub fn add_common_api<Author, GetterExecutor, AccessShieldingKey, OcallApi, Stat
 
 	io_handler.add_sync_method("state_getRuntimeVersion", |_: Params| {
 		debug!("worker_api_direct rpc was called: state_getRuntimeVersion");
-		let parsed = "world";
-		Ok(Value::String(format!("hello, {}", parsed)))
+		let runtime_version = VERSION;
+
+		Ok(json!({
+			// "specName": runtime_version.spec_name,
+			// "implName": runtime_version.impl_name,
+			"authoringVersion": runtime_version.authoring_version,
+			"specVersion": runtime_version.spec_version,
+			"implVersion": runtime_version.impl_version,
+			"apis": runtime_version.apis,
+			"transactionVersion": runtime_version.transaction_version,
+			"stateVersion": runtime_version.state_version,
+		}))
```

Comment:
What's a better way of achieving this for [`RuntimeVersion`](https://docs.rs/sp-runtime/7.0.0/sp_runtime/enum.RuntimeString.html)? 

I see that it implements a `serde::Serialize` trait:

https://github.com/paritytech/substrate/blob/033d4e86cc7eff0066cd376b9375f815761d653c/primitives/version/src/lib.rs#L159

and has this `Display` implementation:

https://github.com/paritytech/substrate/blob/033d4e86cc7eff0066cd376b9375f815761d653c/primitives/version/src/lib.rs#L261

cc @kziemianek

Reply:
Hmm it seems that all rpc return value is scale & hex encoded (using `to_hex()`), see other methods.

Maybe it's better to follow this to be consistent

Reply:
Done in https://github.com/litentry/litentry-parachain/pull/3040/commits/75991ca658c6fd1f74b8515856b9b08cff997f90